src/pages/commits/InactiveCommits/index.tsx

```diff
@@ -71,7 +73,16 @@ export const InactiveCommitButton: React.FC<InactiveCommitsProps> = ({
         align="bottom"
         justify="middle"
         trigger={
-          <ButtonContainer role="button">
+          <ButtonContainer
+            onClick={() => {
+              sendEvent({
+                name: "Toggle commit chart label tooltip",
+                type: tooltipType,
+                toggle: tooltipRef?.current ? "close" : "open",
```

Comment:
This logic is confusing why is this ref necessary?

Reply:
The ref is placed on the tooltip body which exists when it's open and is null when it's closed. The alternative to this solution would have been converting the tooltip from controlled to uncontrolled which would actually add more complexity and state change directions since we would have to control the outside click with another event handler. Ideally the tooltip component exposes it's state in the onClick callback.

Reply:
in this case i think it would be better to not log the toggle. Imo the additional complexity isn't worth the extra detail.